sycl/source/detail/memory_manager.cpp

```diff
@@ -500,6 +500,21 @@ void MemoryManager::fill_usm(void *Mem, QueueImplPtr Queue, size_t Length,
   }
 }
 
+void MemoryManager::prefetch_usm(void *Mem, QueueImplPtr Queue, size_t Length,
+                                std::vector<RT::PiEvent> DepEvents,
+                                RT::PiEvent &OutEvent) {
+  sycl::context Context = Queue->get_context();
+
+  if (Context.is_host()) {
+    // TODO: Potentially implement prefetch on the host.
```

Comment:
We have some implementation of prefetch for host already. ./source/detail/platform_util.cpp:123.
Haven't check if it's suitable here.